api/include/opentelemetry/logs/logger.h

```diff
@@ -35,89 +41,118 @@ namespace logs
  **/
 class Logger
 {
+protected:
+  // Default values to set for fields that the user doesn't specify
+  Severity default_severity_;        // default 0 (severity of kInvalid)
+  nostd::string_view default_name_;  // default 0 length, nullptr data
+  nostd::string_view default_body_;  // default 0 length, nullptr data. TODO: currently a simple
+                                     // string, but should be changed "Any" type
+  trace::TraceId default_trace_id_;  // default 00000000000000000000000000000000, in base 16
+  trace::SpanId default_span_id_;    // default 0000000000000000, in base 16
+  trace::TraceFlags default_trace_flags_;  // default 00, in base 16
```

Comment:
We should get rid of this. I think we can specify `""` whereever `default_name_` or `default_body_` is used, and we could could use `{}` instead of `default_trace_id_`, `default_span_id_` and `default_trace_flags_`.

Reply:
Sure, that sounds pretty intuitive - didn't know `{}` could be a value for trace/spanid :') 

Would you say it's better a default Severity be left as is (to `kInvalid=0`) or directly to `kInfo` as default? Would there be any benefits of knowing when the user intentionally set the severity to `kInfo` and when it was overwritten by the API overloads / SDK?